### ISSUE TITLE
Fail to parse hsl(89.99999999999999,100%,50%)

### ISSUE THREAD

**Issue by jean343** [styled-components/polished#343]
- `polished` version: 1.9.3
- `JSS-in_CSS` library and version: none
- Any relevant JS library and version: Node 8

Mixin/Helper/Shorthand you were using and how you were using it:

```javascript
var polished = require("polished")
polished.parseToHsl("hsl(90.999999, 100%, 50%)");
```

What you are seeing:
```
Couldn't parse the color string. Please provide the color as a string in hex, rgb, rgba, hsl or hsla notation.
```

What you expected to see:
I was expecting decimals in hue to be parsed as numbers. I am passing color strings from another library, and they are in the form: `hsl(90.999999, 100%, 50%)`.

Reproduction:
https://runkit.com/embed/cd7ptn0l2jyl

Thanks!!

**Comment by bhough** [comment]
@jean343 thanks for the issue. We were able to confirm it and are currently working on a fix now that 2.0 is out.

**Comment by jean343** [comment]
Thanks!!
